core/toolbox/category.js

```diff
@@ -697,7 +701,7 @@ Blockly.ToolboxCategory.prototype.getChildToolboxItems = function() {
  * Updates the contents to be displayed in the flyout.
  * If the flyout is open when the contents are updated, refreshSelection on the
  * toolbox must also be called.
- * @param {!Blockly.utils.toolbox.ToolboxDefinition} contents The contents
+ * @param {!Blockly.utils.toolbox.FlyoutDefinition} contents The contents
```

Comment:
If `FlyoutDefinition` doesn't include `string`, this needs an explicit `|string`

Reply:
done